dev/release/VERIFY.md

```diff
@@ -50,15 +50,28 @@ You can install them by the followings on Debian GNU/Linux and Ubuntu:
 You can install them by the followings on CentOS 7:
 
 ```console
-% sudo yum install -y gobject-introspection-devel
+% sudo yum install -y gobject-introspection-devel git curl
+% cd /etc/pki/rpm-gpg
+% curl -O http://springdale.princeton.edu/data/springdale/7/x86_64/os/RPM-GPG-KEY-springdale
+% cd /etc/yum.repos.d
+% cat << EOF > Springdale-SCL.repo
+[Springdale-SCL]
+name=Springdale - SCL
+baseurl=http://springdale.princeton.edu/data/springdale/SCL/$releasever/$basearch
+gpgcheck=1
+enabled=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-springdale
+EOF
+% source scl_source enable devtoolset-11
```

Comment:
What is `scl_enable`? It seems that this file doesn't exist in normal CentOS 7.

Reply:
This enables a software collection without creating a new bash shell see:
- https://unix.stackexchange.com/questions/175851/how-to-permanently-enable-scl-centos-6-4
- https://www.softwarecollections.org/en/docs/guide/#sect-Enabling_Support_for_Software_Collections
If it `scl_enable` is not available may need to add
```
yum install scl-utils 
```

Reply:
Ah, sorry. I misunderstood the argument name (`scl_enable`).

We don't need to use `source` here because `scl_source` is in `PATH`.